NDTensors/src/lib/Sectors/src/category_definitions/su.jl

```diff
@@ -64,20 +79,96 @@ end
 
 #
 # Specializations for the case SU{2}
-# Where irreps specified by dimension "d"
+# Where irreps specified by quantum_dimension "d"
 #
 
-dimension(s::SU{2}) = 1 + label(s)[1]
+# SU2 is an alias for SU{2}
+const SU2 = SU{2}
```

Comment:
I think it is better to define it like this:
```suggestion
const SU2{T} = SU{2,T,1}
```
so you can take advantage of the type parameter `T` as needed.